docs/classic/backgammon.md

```diff
@@ -2,16 +2,112 @@
+| 6         | WHITE - 2nd point, 3rd component                                    | 0.0  | 1.0  |
+| 7         | WHITE - 2nd point, 4th component                                    | 0.0  | 6.0  |
+| ...       |                                                                     |      |      |
+| 92        | WHITE - 24th point, 1st component                                   | 0.0  | 1.0  |
+| 93        | WHITE - 24th point, 2nd component                                   | 0.0  | 1.0  |
+| 94        | WHITE - 24th point, 3rd component                                   | 0.0  | 1.0  |
+| 95        | WHITE - 24th point, 4th component                                   | 0.0  | 6.0  |
+| 96        | WHITE - BAR checkers                                                | 0.0  | 7.5  |
+| 97        | WHITE - OFF bar checkers                                            | 0.0  | 1.0  |
+| 98        | BLACK - 1st point, 1st component                                    | 0.0  | 1.0  |
+| 99        | BLACK - 1st point, 2nd component                                    | 0.0  | 1.0  |
+| 100       | BLACK - 1st point, 3rd component                                    | 0.0  | 1.0  |
+| 101       | BLACK - 1st point, 4th component                                    | 0.0  | 6.0  |
+| ...       |                                                                     |      |      |
+| 190       | BLACK - 24th point, 1st component                                   | 0.0  | 1.0  |
+| 191       | BLACK - 24th point, 2nd component                                   | 0.0  | 1.0  |
+| 192       | BLACK - 24th point, 3rd component                                   | 0.0  | 1.0  |
+| 193       | BLACK - 24th point, 4th component                                   | 0.0  | 6.0  |
+| 194       | BLACK - BAR checkers                                                | 0.0  | 7.5  |
+| 195       | BLACK - OFF bar checkers                                            | 0.0  | 1.0  |
+| 196 - 197 | Current player                                                      | 0.0  | 1.0  |
+
+Encoding of a single point (it indicates the number of checkers in that point):
+
+| Checkers | Encoding                                |           
+| -------- | --------------------------------------- |
+| 0        | [0.0, 0.0, 0.0, 0.0]                    |
+| 1        | [1.0, 0.0, 0.0, 0.0]                    |
+| 2        | [1.0, 1.0, 0.0, 0.0]                    |
+| >= 3     | [1.0, 1.0, 1.0, (checkers - 3.0) / 2.0] |
+
+Encoding of BAR checkers:
+
+| Checkers | Encoding             |           
+| -------- | -------------------- |
+| 0 - 14   | [bar_checkers / 2.0] |
+
+Encoding of OFF bar checkers:
+
+| Checkers | Encoding              |           
+| -------- | --------------------- |
+| 0 - 14   | [off_checkers / 15.0] |
+
+Encoding of the current player:
+
+| Player  | Encoding   |           
+| ------- | ---------- |
+| WHITE   | [1.0, 0.0] |
+| BLACK   | [0.0, 1.0] |
+
+#### Action Space
```

Comment:
Does this description help at all? 

Each action value encodes the two points to move checkers from (source locations), and which dice roll to use first. A full action moves the first checker from the first source location forward by the amount of the first dice roll (either low roll or high roll, depending on the action), and then a checker from the second source location forward by the amount of the other dice roll.

Reply:
What's a full action?

Reply:
A full action is like both moves that the agent makes. But I think it's a confusing term so I tried to avoid it. Is this more clear?

An agent's turn involves rolling two dice and then using the values of those two rolls to move checkers from one point to another, or move checkers off of the board (bear off).
 
#### Action Encoding

Each action value encodes the two points to move checkers from (source locations), and which dice roll to use first. An action moves a checker from the first source location forward by the amount of the first dice roll (either low roll or high roll, depending on the action value), and then moves a checker from the second source location forward by the amount of the other dice roll.